### PR TITLE
publish coordinator

### PR DESCRIPTION
**Pull request by dheniges**
- It versions a document and copies all its fields and values
- Coordinator tests are integration because I was concerned I was just making up stuff and the publish wasn't actually happening 🤷‍♂  I can add unit tests as well if that sounds useful
- DocumentVersion gets Business-y

### CODE REVIEW

File: api/src/businesstime/documentversion.test-i.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    it('should belong to the document', () => {
+      expect(documentVersion.docId).toBe(factoryDocument.id)
+    })
+
+    it('should belong to the org', () => {
+      expect(documentVersion.orgId).toBe(factoryDocument.orgId)
+    })
+
+    it('should have a verison number', () => {
```
Comment (jjidt) [inline api/src/businesstime/documentversion.test-i.js]: typo

File: api/src/coordinators/documentVersion.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import BusinessDocument from '../businesstime/document'
+import BusinessField from '../businesstime/field'
+import BusinessDocumentVersion from '../businesstime/documentversion'
+
+const publishDocumentVersion = async function(docId, projectId, orgId) {
+  const doc = await BusinessDocument.findByIdForProject(docId, projectId, orgId)
```
Comment (jjidt) [inline api/src/coordinators/documentVersion.js:7]: do we need a check here that the document exists and a 404 if not? might be coming back null

Comment (jjidt) [reply api/src/coordinators/documentVersion.js:7]: probably a test for that situation too

**Review by jjidt — CHANGES_REQUESTED** [review]
I like the integration tests here, and I don't think we need unit tests as long as we have these  💯 

just a couple small changes

**Comment by dheniges** [comment]
@jjidt updated

**Review by jjidt — APPROVED** [review]

**Pull request merged**